web/templates/topic_edit_form.hbs

```diff
@@ -1,9 +1,10 @@
 {{! Client-side Handlebars template for rendering the topic edit form. }}
 
 <form class="topic_edit_form">
-    <input type="text" value="" autocomplete="off" maxlength="{{ max_topic_length }}"
-      class="inline_topic_edit header-v {{#unless realm_mandatory_topics}}empty-topic-placeholder-display{{/unless}}"
-      {{#unless realm_mandatory_topics}}placeholder="{{empty_string_topic_display_name}}"{{/unless}} />
+    <input type="text" value="" autocomplete="off" maxlength="{{ max_topic_length }}" class="inline_topic_edit header-v"/>
+    <span id="inline-topic-edit-placeholder" class="placeholder">
```

Comment:
Can't use `ID` here, since I think you can have more than one of these open at a time.

Reply:
changed it to class.